src/client/v2/indexer/lookupAssetTransactions.ts

```diff
@@ -61,7 +61,7 @@ export default class LookupAssetTransactions extends JSONRequest {
    *        .do();
    * ```
    *
-   * @param type - one of `pay`, `keyreg`, `acfg`, `axfer`, `afrz`, `appl`, `stpf`
```

Comment:
I should say I'm not confident about my previous question. I could see an argument for keeping this codec. Just wanted to provoke a thought or two.

Reply:
I don't think that a state proof txn would even have an associated asset, so in that sense lookupAssetTransactions would never return a state proof in the response and so the filter would be a no-op.

That's my understanding of it currently.